src/ITILSolution.php

```diff
@@ -302,7 +302,11 @@ public function post_addItem()
 
         // Handle rich-text images and uploaded documents
         $this->input["_job"] = $this->item;
-        $this->input = $this->addFiles($this->input, ['force_update' => true]);
+        $options = ['force_update' => true];
+        if (!isset($this->input['_filename']) && isset($this->input['_content'])) {
+            $options['name'] = 'content';
+        }
```

Comment:
IMHO, it should be possible to do it automatically directly in the `CommonDBTM::addFiles()` method.
```diff
        $default_options = [
            'force_update'  => false,
            'content_field' => 'content',
-            'name'          => 'filename',
+            'name'          => array_key_exists('_filename', $input) ? 'filename' : ($options['content_field'] ?? 'content'),
            'date'          => null,
        ];
```

There are many calls to `CommonDBTM::addFiles()` that could be simplified by removing the custom `name` option (the `name` option could even be removed, but not in the `10.0/bufgixes` branch).